.github/workflows/test.yml

```diff
@@ -30,15 +37,18 @@ jobs:
           python-version: ${{ matrix.python }}
       - name: Install
         run: |
-          python -m pip install --upgrade pip setuptools wheel tox
+          python -m pip install --upgrade pip setuptools wheel tox tox-gh-actions
       - name: Pyright (Python ${{ matrix.python }})
         run: tox -e pyright -- --pythonversion ${{ matrix.python }}
 
   test-sqlite:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python: ["3.11"]
```

Comment:
Worth keeping `3.12` here now that it's supported [since 4.2.8](https://docs.djangoproject.com/en/4.2/releases/4.2/#python-compatibility) ?